app/components/NextEventInfo/NextEventInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import type { MeetupEvent } from "~/models/meetup";
+import { DateTime } from "luxon";
+import { isEmptyString } from "~/utils";
+import MeetupButton from "~/components/MeetupButton";
+
+const DEFAULT_VENUE: MeetupEvent["venue"] = {
+  name: "H-E-B Digital & Favor Eastside Tech Hub (and online)",
+  address: "2416 E 6th St",
+  city: "Austin",
+  state: "TX",
+};
+
+const CENTRAL_TIMEZONE = "America/Chicago" as const;
+
+function isUndefinedVenue(venue: MeetupEvent["venue"]): boolean {
+  return (
+    isEmptyString(venue.address) ||
+    isEmptyString(venue.city) ||
+    isEmptyString(venue.state)
+  );
+}
+
+export default function NextEventInfo({ event }: { event: MeetupEvent }) {
+  const date = DateTime.fromISO(event.dateTime).setZone(CENTRAL_TIMEZONE);
```

Comment:
Do you mind using [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) since it's built into the browser?

Luxon is a [pretty big](https://bundlephobia.com/package/luxon@3.1.1) library just for this one utility (it may tree shake well, I'm not sure)

Reply:
Also, I don't think we need to set the timezone, since the ISO string should have it built in and we want to display to the users the time in their timezone. Perhaps I'm not thinking about this correctly though

Reply:
since we're doing SSR, the user's timezone becomes tricky. when we SSR, the timezone will be UTC. the browser will be something else. and thus you now have to deal with the blip: SSR render in UTC but the hydration shifts it to a diff time zone. (i also think that react will give you a warning/error that the content is different. but it's a bit annoying since you won't see it during local dev.) 

[there are solutions](https://edgebound.dev/the-pre-rendering-blip). likely rendering a default timezone via SSR and let the client do a CSS animation into the browser's time zone.

on a related but different note, i think a physical, in-person event is a scenario where it makes sense to _not_ show the time in the browser's timezone. and use the event's timezone.

and thus i elected to forgo the blip altogether and render in the event's timezone.

Reply:
Nice self plug 👀 

Right now I'd rather avoid doing any client-side hydration, since we really don't need JS (yet) #40 

> on a related but different note, i think a physical, in-person event is a scenario where it makes sense to not show the time in the browser's timezone. and use the event's timezone.

I think you're totally right, which is what I came here to say. Just saying "CST" no matter the user's time zone seems like the right way to go, so I'm on board with your solution 💪